atomic_reactor/plugins/pre_bump_release.py

```diff
@@ -164,3 +166,20 @@ def run(self):
         self.log.info("setting %s=%s", release_label, next_release)
         # Write the label back to the file (this is a property setter)
         dockerfile_labels[release_label] = next_release
+
+        if not self.predeclare_build:
+            return
+
+        nvr_data = {
+            'name': component,
+            'version': version,
+            'release': dockerfile_labels[release_label]
+        }
+
+        try:
+            pre_declared = self.xmlrpc.CGInitBuild(PROG, nvr_data)
+        except Exception:
+            raise RuntimeError('unable to pre-declare build {}'.format(nvr_data))
```

Comment:
Might be nice to `log.exception()` here so we can see what the problem was.

Reply:
I'd go even further and report reason to koji output
```
except Exception as e:
    raise RuntimeError('unable to pre-declare build {}: {}'.format(nvr_data, e))
```

Reply:
raising now with full exc error

Reply:
Looking at how CGImport does this, it's different. There, it logs information about what it submitted, and re-raises the exception. The equivalent here would be:
```
log.debug('unable to pre-declare build {}'.format(nvr_data))
raise
```

Not sure which is better, but it would be nice if they were consistent.

Reply:
ok I'll do the same what we have for cgimport

log data we've submitted and re-raise error